pkg/api/unversioned/types.go

```diff
@@ -308,6 +308,15 @@ type APIVersions struct {
 	TypeMeta `json:",inline"`
 	// versions are the api versions that are available.
 	Versions []string `json:"versions"`
+	// a map of client CIDR to server address that is serving this group.
+	// Clients can use the appropriate server address as per the CIDR that they match.
+	// In case of multiple matches, clients should use the longest matching CIDR.
+	// The server returns only those CIDRs that it thinks that the client can match.
+	// For ex: the master will return an internal IP CIDR only, if the client reaches the server using an internal IP.
+	// Server looks at X-Forwarded-For header or X-Real-Ip header or request.RemoteAddr (in that order) to get the client IP.
+	ServerAddressByClientCIDRMap []ServerAddressByClientCIDR `json:"serverAddressByClientCIDRMap"`
+	// Optional path prefix that is inserted between the address and the group name to reach the group endpoint.
+	PathPrefix string `json:"pathPrefix,omitempty"`
```

Comment:
@smarterclayton and I agreed elsewhere in this PR that this only needs to be in the summarizer's config, not here?